library/psa_crypto_se.c

```diff
@@ -148,17 +148,23 @@ psa_status_t psa_load_se_persistent_data(
 {
     psa_status_t status;
     psa_storage_uid_t uid;
+    size_t length;
 
     status = psa_get_se_driver_its_file_uid( driver, &uid );
     if( status != PSA_SUCCESS )
         return( status );
 
+    /* Read the amount of persistent data that the driver requests.
```

Comment:
While the section below explains the code, this one rather describes the function behavior. I find it fine, but instead of placing two such comment sections next to each other, the first one could be placed as a doxygen note?

Reply:
What do you mean by “placed as a doxygen note”? This implements the behavior documented in `crypto_se_driver.h` for the `persistent_data` of `psa_drv_se_context_t`, which `driver->internal.persistent_data` accesses. I think it's useful to explain what's going on at the point of implementation.

Reply:
I may have misunderstood the comment to be information that may be interesting to the function's user and as such placed in the doxygen comment for the function. I found it suspicious for a function to have the same amount of comment lines that it has code, so I suggested revising it, which you kindly have. Thanks.